### PR TITLE
modernize gems and dependencies.

### LINKED ISSUE #50: Sass deprecated soon
This gem has dependency sass_rails which use sass too. Ruby Sass is deprecated and will be unmaintained as of 26 March 2019. Will you update dependency to use sassc_rails? Thank you.

[http://sass.logdown.com/posts/7081811 ](http://sass.logdown.com/posts/7081811)

### PR DESCRIPTION
**Pull request by senny**
Closes #50

This patch replaces the deprecated `sass-rails` gem with
`sassc-rails`. It  also modernizes the testing dependencies and now
uses a headless chrome driver (`apparition`) instead of `poltergeist`.

### CODE REVIEW

**Review by siegy22 — APPROVED** [review]
LGTM, will you create the release once merged? 😄

**Comment by iamghazi** [comment]
Please merge and release this. I am getting dependency issues.

**Comment by dwightwatson** [comment]
Any chance of a tagged release too?

**Comment by siegy22** [comment]
@dwightwatson I'm still working on updating PDF.js too, will release it after I'm done.

**Pull request merged**
